docs/adrs/006-kdm-strategy.md

```diff
@@ -0,0 +1,31 @@
+# 6. KDM Branching Strategy for RKE2
+
+Date: 2023-09-14
+
+## Status
+
+New
+
+## Context
+
+Rancher Manager uses [KDM](https://github.com/rancher/kontainer-driver-metadata/) for its integration of RKE2. This manages the following behaviors for each RKE2 version:
```

Comment:
Should the ADR go here, in KDM, or ecm-distro-tools? This feels like an overarching issue where we want to observe parity for both K3s and RKE2, and the change were discussing doesn't really affect the distro - so I don't feel like it belongs in this repo.